packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -614,3 +615,60 @@ export interface GetIndexerMakerStatisticsResponse {
   rewardCoefficient: BigDecimal;
   makers: IndexerMaker[];
 }
+
+export interface GetIndexerLeaderboardParams {
+  contestId: number;
+  rankType: IndexerLeaderboardRankType;
+  // Min rank inclusive
+  startCursor?: string;
+  limit?: number;
+}
+
+export interface IndexerLeaderboardParticipant {
+  subaccount: string;
```

Comment:
Shouldn't this be processed into `Subaccount`?

Reply:
Yup, will fix

Reply:
addressed!